src/Utils/utils.ts

```diff
@@ -561,3 +562,26 @@ export function omitBy<T extends Record<string, unknown>>(
     Object.entries(obj).filter(([_, value]) => !predicate(value)),
   ) as Partial<T>;
 }
+
+export const copyToClipboard = async (
+  text: string,
+  successMsg: string,
+  errorMsg: string,
+  onSuccess: () => void,
+  onResetIcon: () => void,
+  resetIconDuration = 2500,
+) => {
+  if (!text) {
+    Notification.Error({ msg: "Nothing To Copy" });
+    return;
```

Comment:
let's not show this icon and call this function in the first place if there's nothing to copy.